startup.sh

```diff
@@ -71,6 +71,13 @@ function install_docker() {
   yum install -y docker-ce docker-ce-cli containerd.io
 }
 
+function install_docker_rhel() {
+  subscription-manager repos --enable=rhel-7-server-rpms
+  subscription-manager repos --enable=rhel-7-server-extras-rpms
+  subscription-manager repos --enable=rhel-7-server-optional-rpms
```

Comment:
should we disable repos at the end?

Reply:
I wouldn't recommend disabling a repo because it could break yum update later

Reply:
ok